app/common/lib/updateUtil.js

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const Immutable = require('immutable')
+const updateStatus = require('../../../js/constants/updateStatus')
```

Comment:
I would leave `updateStatus` in `constants` folder, because that's what we have there. What we can do is rename this file and move it into `state` folder. What do you think?